__tests__/app.test.js

```diff
@@ -0,0 +1,112 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("GET /api/healthcheck", () => {
```

Comment:
This endpoint doesn't exist in this project. The following test shows us that when we go to /api/topics we get a 200, so this test can be removed 👍